morello/morello.md

```diff
@@ -1,13 +1,13 @@
-# Morello Supplement to the Arm C Language Extensions
-
-01alpha
-
-Date of Issue: 02 July 2021
-
-![image](Arm_logo_blue_RGB.svg)
-
-------------------------------------------------------------------------
-
+---
+layout: default
+permalink: /morello/morello.html
+title: Morello Supplement to the Arm C Language Extensions
+version: 01alpha
+date-of-issue: 02 July 2021
+---
```

Comment:
This doesn't render nicely anymore when browsed on GitHub: https://github.com/ValeriaDaneva/acle/blob/github.io-web-UI/morello/morello.md

But we are happy with it because we will be browsing these specs via GutHub.io. :)

One questions I have: is there a way to make `title`,`version`, and `date-of-issue` shared with the ones defined in the YAML file for the PDF rendering? It would be better to change these information in just once place when needed...

Reply:
Not that I'm aware of? I suppose we can add one file for the three parameters and then append them to the other two in the bash script. Or, perhaps just add all the yaml info at the top of each file since we're using it for both pdf and html generation anyway.

Reply:
I am starting to think that we should merge all the YAML content (pdf and html one) in the header of the md file of the specs, and remove the mechanisms that uses the `<!---END_OF_HTML_HEADER--->`. Teh variables that are not used in the HTML will be ignored in the PDF, and viceversa, but the variable in commn will stay the same. At the end of the day, we used the `<!---END_OF_HTML_HEADER--->` mechanism to make sure we coudl still have the Arm logo when looking at the pages on github.io, but given that we are replacing the browser experience on github with the one on github.io via Pages, I don't see an issue in getting rid of such mechanism, as we see the raw YAML content anyway when browsing the content of the repository.

We will do this merging of the PDF in a separate patch though. Keep the `<!---END_OF_HTML_HEADER--->` mechanism for now.